software/trieve.yaml

```diff
@@ -0,0 +1,15 @@
+name: Trieve
+website_url: https://trieve.ai
+description: All-in-one RESTful RAG, search, recommendations, and analytics engine.
+licenses:
+  - BUSL-1.1
+platforms:
+  - Rust
+  - Docker
+tags:
+  - Search Engines
+source_code_url: https://github.com/devflowinc/trieve
+demo_url: https://hn.trieve.ai
+stargazers_count: 1352
+updated_at: "2024-09-12"
+archived: false
```

Comment:
```suggestion
```
this metadata will be automatically maintained, no need for this 👍🏻

Reply:
Awesome, that makes more sense. Fixed!